pkg/controller/secret/secret_controller_test.go

```diff
@@ -174,6 +174,7 @@ func Test_addPDSecretToAlertManagerConfig(t *testing.T) {
 		Description:    `{{ .CommonLabels.alertname }} {{ .CommonLabels.severity | toUpper }} ({{ len .Alerts }})`,
 		Details: map[string]string{
 			"link":         `{{ if .CommonAnnotations.link }}{{ .CommonAnnotations.link }}{{ else }}https://github.com/openshift/ops-sop/tree/master/v4/alerts/{{ .CommonLabels.alertname }}.md{{ end }}`,
+			"link2":        `{{ if .CommonAnnotations.runbook }}{{ .CommonAnnotations.runbook }}{{ else }}{{ end }}`,
```

Comment:
Is the config happy with an empty value in `link2`?  Does it become an empty string or just nothing?

Reply:
I think it should be OK, just yaml at the end, value would be `null`.  I'll approve and we can just confirm in stage with some real alerts.